### DISCUSSION TITLE
Be Able To Have A different style

### DISCUSSION THREAD

**Discussion by rovarkamil** [mskocik/svelecte#173] [Ideas]
great job at creating such a great component but i have on more tip: cannot change the style without resetting all of the style and cannot style the dropdown box. so if you put the className as a different variable for it's style and have it's class as it's  default style variable i would really appreciate that. thank again for such a great component.

**Comment by mskocik** [comment]
Yes, styles could use little update, I agree. I am planning it for v4

**Reply by mskocik** [reply]
v4 released last week, so it should be easy to restyle now :)

**Reply by woss** [reply]
i chose this lib over all others because it seems to be a lot better than the others, thing is that I am missing the examples how to style it with the tailwind. would you be interested in providing such example?
